### PR TITLE
Separate integrations section of docs

### PR DESCRIPTION
**Pull request by bracesproul**
Todo in future PR:
Add script for generating llm method table (like in py)
Move how-to's out of folders and into top level modules

### CODE REVIEW

File: docs/docs/integrations/chat/index.mdx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+sidebar_position: 1
+sidebar_class_name: hidden
+---
+
+# Chat models
+
+## Features (natively supported)
+
+All ChatModels implement the Runnable interface, which comes with default implementations of all methods, ie. `invoke`, `batch`, `stream`, `stream`. This gives all ChatModels basic support for invoking, streaming and batching, which by default is implemented as below:
```
Comment (jacoblee93) [inline docs/docs/integrations/chat/index.mdx:10]: Did you mean `streamLog` maybe?

File: docs/docs/integrations/chat_memory/index.mdx
Comment (jacoblee93) [inline docs/docs/integrations/chat_memory/index.mdx:1]: What happened here?

File: docs/docs/integrations/llms/index.mdx [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+sidebar_class_name: hidden
+---
+
+# LLMs
+
+## Features (natively supported)
+
+All LLMs implement the Runnable interface, which comes with default implementations of all methods, ie. `invoke`, `batch`, `stream`, `map`. This gives all LLMs basic support for invoking, streaming, batching and mapping requests, which by default is implemented as below:
+
+- _Streaming_ support defaults to returning an `AsyncIterator` of a single value, the final result returned by the underlying LLM provider. This obviously doesn't give you token-by-token streaming, which requires native support from the LLM provider, but ensures your code that expects an iterator of tokens can work for any of our LLM integrations.
```
Comment (jacoblee93) [inline docs/docs/integrations/llms/index.mdx:12]: Explain `map` too?

File: langchain/package.json [hunk 3]
```diff
@@ -926,6 +926,7 @@
     "sqlite3": "^5.1.4",
     "srt-parser-2": "^1.2.2",
     "ts-jest": "^29.1.0",
+    "ts-morph": "^20.0.0",
```
Comment (jacoblee93) [inline langchain/package.json:929]: What's `ts-morph`?

**Review by jacoblee93 — COMMENTED** [review]
Thanks so much for this! Will kick off a preview build in a bit.

File: docs/vercel.json [hunk 4]
```diff
@@ -359,6 +359,46 @@
...
+      "source": "/docs/modules/memory/integrations(/.*)?",
+      "destination": "/docs/integrations/chat_memory$1"
+    },
+    {
+      "source": "/docs/modules/agents/tools/integrations(/.*)?",
+      "destination": "/docs/integrations/tools$1"
+    },
+    {
+      "source": "/docs/modules/agents/toolkits(/.*)?",
+      "destination": "/docs/integrations/toolkits$1"
```
Comment (jacoblee93) [inline docs/vercel.json:401]: Nice

**Review by jacoblee93 — COMMENTED** [review]
Thanks so much for this! Will kick of a preview build in a bit.

File: docs/docs/integrations/chat/index.mdx (hunk 1 above)
Comment (bracesproul) [reply docs/docs/integrations/chat/index.mdx:10]: Ah no meant batch, thanks for catching

**Pull request merged**